relnotes

```diff
@@ -243,30 +243,23 @@ ci_job_status () {
   local red=${TPUT[RED]}
   local green=${TPUT[GREEN]}
   local off=${TPUT[OFF]}
-  local extra_flag
+  local url="https://dl.k8s.io/ci/latest"
 
   if ((FLAGS_htmlize_md)); then
     red="<FONT COLOR=RED>"
     green="<FONT COLOR=GREEN>"
     off="</FONT>"
   fi
 
-  # If working on a release branch assume --official for the
-  # purpose of displaying find_green_build output
   if [[ $CURRENT_BRANCH =~ release- ]]; then
-    extra_flag="--official"
-  else
-    # For master branch, limit the analysis to 30 primary ci jobs.
-    # This is necessary due to the recently expanded blocking test list for
-    # master.  The expanded test list is often unable to find a complete
-    # passing set and find_green_build runs unbounded for hours.
-    extra_flag="--limit=30"
+    url="$url-${CURRENT_BRANCH/release-/}"
   fi
+  url="$url.txt"
 
   # State of tree
   echo
   echo "## State of $CURRENT_BRANCH branch"
-  if content=$(find_green_build -v $extra_flag $CURRENT_BRANCH); then
```

Comment:
This does more than just printing the version, but it also prints a summary of which tests have been failing, skipped, ...

I am not sure if we necessarily need all that info, but I would like us to dig in a bit about:
- which of that data we'd still like to have
- how/where else we can get it

As this is mostly used by @kubernetes/patch-release-team , what do you all think?

Reply:
agreed. I wonder if the `--buildversion` suggestion comes from here.